SimplifiedTests/NYPLAxisTests/NYPLAxisXMLTests/DummyXML.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0"?>
```

Comment:
can you rename this file DummyCatalog.xml instead?

Reply:
Done